### PR TITLE
Smoothing scroll effect for navigation

### PR DESCRIPTION
**Pull request by tanaphonble**
ทำให้ scroll ดู smooth ขึ้นครับ

### CODE REVIEW

File: components/NavigationBar.js [hunk 1]
```diff
@@ -19,11 +20,12 @@ export default function NavigationBar () {
   )
 }
 
-function NavigationLink ({ href, disabled, children }) {
+function NavigationLink({ href, disabled, children }) {
   return (
-    <span className={disabled ? 'disabled' : ''}>
-      <LevitatingLink href={href}>{children}</LevitatingLink>
-      <style jsx>{`
+    <Link to={children.toLowerCase()} smooth={true} duration={500} >
```
Comment (dtinth) [inline components/NavigationBar.js:25]: Using a `Link` component from `react-scroll` means that the user cannot Cmd+Click to open the link in a new tab.

File: pages/index.js [hunk 2]
```diff
@@ -5,17 +5,26 @@ import SectionSeparator from '../components/SectionSeparator'
...
 import TicketsSection from '../components/TicketsSection'
 import SponsorsSection from '../components/SponsorsSection'
+import { Element } from 'react-scroll'
 
 export default () => (
   <MainLayout>
     <HeroUnit />
     <SectionSeparator first />
-    <section id='tickets'><TicketsSection /></section>
+    <Element name='tickets'>
```
Comment (dtinth) [inline pages/index.js:14]: The page should not be aware of the scrolling mechanism.

Ideally, only one file should be changed to allow smooth scrolling.

File: components/NavigationBar.js [hunk 3]
```diff
@@ -19,11 +20,12 @@ export default function NavigationBar () {
...
+      <span className={disabled ? 'disabled' : ''}>
+        <LevitatingLink href={href}>{children}</LevitatingLink>
```
Comment (dtinth) [inline components/NavigationBar.js:27]: Nesting a LevitatingLink inside a Link yields a React warning:

```
Warning: validateDOMNesting(...): <a> cannot appear as a descendant of <a>. See NavigationLink > _ > Link > a > ... > LevitatingLink > a.
```

Comment (dtinth) [reply components/NavigationBar.js:27]: Also, the smooth scrolling doesn’t seem to work, and the disabled items is not displaying properly. Here is a screenshot in Google Chrome.

image

**Review by dtinth — COMMENTED** [review]
Thanks for your participation!

However, this issue has already been solved by #81. Please see that pull request for the correct solution.

Following is a review of your PR.

**Comment by dtinth** [comment]
Because this issue is already solved, I will have to close this PR. Thank you for your understanding.

If you could improve this website in some other way, please feel free to send us another pull request. 😃

**Comment by tanaphonble** [comment]
Thank you very much for reviewing my PR.
I will learn from this. :)

**Pull request closed without merging**